cmake/common/gtest.cmake

```diff
@@ -101,7 +101,7 @@ macro(add_gtest test)
 
             # Search for google value-parameterized tests instantiations
             unset(GTEST_INSTANTATIONS) # list of instantiations names
-            file(STRINGS ${GTEST_SOURCE_FILE} GTEST_INSTANTIATION_NAMES REGEX ^INSTANTIATE_TEST_CASE_P)
+            file(STRINGS ${GTEST_SOURCE_FILE} GTEST_INSTANTIATION_NAMES REGEX ^INSTANTIATE_TEST_SUITE_P)
```

Comment:
This should be modified?

Reply:
yes, as `INSTANTIATE_TEST_CASE_P` is deprecated, and this line searchs for parametrized tests on given source files